hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/AbstractTestS3AEncryption.java

```diff
@@ -78,6 +78,14 @@ protected void patchConfigurationEncryptionSettings(
       0, 1, 2, 3, 4, 5, 254, 255, 256, 257, 2 ^ 12 - 1
   };
 
+  /**
+   * Skips the tests if encryption is not enabled in configuration.
+   *
+   * @implNote We can use {@link #createConfiguration()} here since
```

Comment:
not sure if maven is set up to use those tags; no reason why we shouldn't start though

Reply:
I never realised they weren't JavaDoc standard tags.

I'm not really fussed either way, I can move to an inline comment really.

Just as a note, if this was in `public` src code then I think builds would fail with something like this:

```
[ERROR] hadoop/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/audit/impl/LoggingAuditor.java:61: error: unknown tag: myTag
[ERROR]  * @myTag What happens if I build this?
```

Happy to move it to inline comment in `trunk` - let me know.